### ISSUE TITLE
NetworkingV1Api object has no attribute create_ingress

### ISSUE THREAD

**Issue by everping** [kubernetes-client/python#1545]
Hi,

I used `utils.create_from_yaml` to create a deployment and got the error `AttributeError: 'NetworkingV1Api' object has no attribute 'create_ingress'`

I attached my code and deployment.yml as below. Please help me to resolve it

```python
from kubernetes import client, config, utils

kube_config = 'kubeconfig.yml'
deployment_file = 'deployment.yml'
config.load_kube_config(config_file=kube_config)
kube_client = client.AppsV1Api()

utils.create_from_yaml(kube_client, yaml_file=deployment_file)
```

```yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: project-test
spec:
  rules:
    - http:
        paths:
          - path: /
            pathType: Prefix
            backend:
              service:
                name: project-test
                port:
                  number: 3000
      host: domain.com
---
apiVersion: v1
kind: Service
metadata:
  name: project-test
  labels:
    app: project-test
spec:
  ports:
    - port: 3000
      protocol: TCP
  selector:
    app: project-test

---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: project-test
  labels:
    app: project-test
spec:
  replicas: 1
  selector:
    matchLabels:
      app: project-test
  template:
    metadata:
      labels:
        app: project-test
    spec:
      containers:
        - name: project-test
          image: registry.digitalocean.com/project-registry/project/test:1234
          volumeMounts:
            - mountPath: /etc/localtime
              name: timezone
          ports:
            - containerPort: 3000
      terminationGracePeriodSeconds: 30
      volumes:
        - name: timezone
          hostPath:
            path: /etc/localtime
      imagePullSecrets:
        - name: do-registry

**Comment by tpalashki** [comment]
I am facing the same problem.

I am also trying to create the ingress by using the create_namespaced_ingress method, but then I get the following 404 (Not Found) error:
```
<html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

My code is similar to:
```python
configuration = client.Configuration()
with client.ApiClient(configuration) as api_client:
    api_instance = client.ExtensionsV1beta1Api(api_client)
    try:
        api_instance.create_namespaced_ingress("default", client.ExtensionsV1beta1Ingress())
    except ApiException as ex:
        print("Exception when calling ExtensionsV1beta1Api->create_namespaced_ingress: %s\n" % ex)
```

I tried with an actual (non-empty) body, but get the same error.

**Comment by HighWatersDev** [comment]
Same issue here. It doesn't seem that the library supports NetworkingV1Api for Ingress. I only see support for network_policy. The latest I see for Ingress is NetworkingV1beta1Api but that doesn't work with latest API version.

**Comment by everping** [comment]
My workaround is using `os.system("kubectl ...")` instead of using that SDK

**Comment by yliaog** [comment]
https://github.com/kubernetes-client/python/tree/b313b5e74f7cf222ccc39d8c5bf8a07502bd6db3/kubernetes/docs

it's correct that currently only NetworkingV1beta1Ingress is supported.

https://kubernetes.io/docs/concepts/services-networking/ingress/ shows that NetworkingV1Ingress is added from 1.19 kubernetes release. this python library is adding 1.19 release as we speak. it is going to be available in a few weeks.

**Comment by servens** [comment]
@yliaog  I see that the beta version of 19 has been released, but the networking v1api still doesn't see create_namespaced_ingress。

kubernetes                    19.15.0a1

**Comment by yliaog** [comment]
https://github.com/kubernetes-client/python/blob/release-19.0/kubernetes/docs/NetworkingV1Api.md#create_namespaced_ingress

it is now available in 1.19.
